app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -146,4 +148,13 @@ class LoginFragment : Fragment() {
             else -> super.onOptionsItemSelected(item)
         }
     }
+
+    private fun showSnackbar() {
+        val textSnackbar = arguments?.getString("snackbar_message")
```

Comment:
Where is this used below?

Reply:
I only used this to check whether snackbar should be showed or not

Reply:
You should show the passed string. Imagine telling the fragment what needs to be shown as snackbar and it ignores your message and shows whatever it wants